cmd/empire/factories.go

```diff
@@ -227,3 +229,21 @@ func newHBReporter(key, env string) (reporter.Reporter, error) {
 	// since it thinks MultiReporter is a struct literal.
 	return append(reporter.MultiReporter{}, empire.DefaultReporter, r), nil
 }
+
+// Auth provider =======================
+
+func newAuthProvider(c *cli.Context) (dockerauth.AuthProvider, error) {
+	provider := dockerauth.NewMultiAuthProvider()
+	provider.AddProvider(dockerauth.NewEcrAuthProvider(ecr.New(newConfigProvider(c))))
+
+	dockerConfigFile, err := os.Open(c.String(FlagDockerAuth))
+	if err == nil {
+		defer dockerConfigFile.Close()
+		dockerConfigProvider, err := dockerauth.NewDockerConfigAuthProvider(dockerConfigFile)
+		if err == nil {
```

Comment:
Aren't there several use cases that do not depend on credentials from Docker config? That is, if the file is not present on disk, it might be intentional. I do see #542, so perhaps there's something I'm missing?